build/README.md

```diff
@@ -94,4 +119,10 @@ Note: this script is not idempotent so if any errors occure, you will need to ru
 
 ```shell
 bundle exec ./release <the-new-version-number> <build | tag | release | changelog | docs | nexus>
-```
\ No newline at end of file
+```
+
+Check the Maven Repo to see if it all worked!
+
+```
+https://repo1.maven.org/maven2/com/zendesk/maxwell/<the-new-version-number>/
```

Comment:
```suggestion
https://repo1.maven.org/maven2/com/zendesk/maxwell/
```

Might be easier, so you don't need to edit the URL (just look for your version on the page)